### PR TITLE
indicator width and height + id and classname (#276 #277)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
```
from taipy.gui import Gui

val = 50

Gui("""
# Test indicator
<|{val}|indicator|value={val}|width=50vw|id=fred|class_name=fred-class|>

<|{val}|indicator|value={val}|orientation=vertical|height=50vh|>
""").run(use_reloader=False)
```

### CODE REVIEW

File: gui/doc/indicator.csv [hunk 1]
```diff
@@ -7,4 +7,6 @@ max,int|float,100,The maximum value of the range.
 format,str,,"The format to use when displaying the value.
 This uses the <code>printf</code> syntax."
 orientation,str,"""horizontal""","The orientation of this slider."
+width,str,None,"the width, in CSS units, of the indicator (used when orientation is default/horizontal)."
```
Comment (FabienLelaquais) [inline gui/doc/indicator.csv]: Capital 'T', please, for consistency.

Drop the 'default/' part.

File: gui/doc/indicator.csv [hunk 2]
```diff
@@ -7,4 +7,6 @@ max,int|float,100,The maximum value of the range.
...
+height,str,None,"the height, in CSS units, of the indicator (used when orientation is vartical)."
```
Comment (FabienLelaquais) [inline gui/doc/indicator.csv]: Capital 'T'.

'vartical' -> 'vertical'

File: gui/doc/indicator.md [hunk 3]
```diff
@@ -51,11 +51,33 @@ The _orientation_ can be specified to "vertical" (or "v") to create a vertical i
     === "Markdown"
 
         ```
-        <|{50}|indicator|orientation=v|value=10|>
+        <|message|indicator|orientation=v|value=10|>
```
Comment (FabienLelaquais) [inline gui/doc/indicator.md:54]: Could this be more explicit?

Like, KPI?

File: gui/src/components/Taipy/Indicator.tsx [hunk 4]
```diff
@@ -73,12 +79,14 @@ const Indicator = (props: IndicatorProps) => {
...
                 },
             }),
-        [horizontalOrientation]
+        [horizontalOrientation, width, height]
     );
 
     return (
         <Tooltip title={hover || ""}>
             <TpSlider
+                id={props.id}
```
Comment (FabienLelaquais) [inline gui/src/components/Taipy/Indicator.tsx:88]: Good catch!

**Review by FabienLelaquais — APPROVED** [review]
With minor doc comments...
Thx!

File: gui/doc/indicator.csv (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/indicator.csv]: done

File: gui/doc/indicator.md (hunk 3 above)
Comment (FredLL-Avaiga) [reply gui/doc/indicator.md:54]: I thought "message" was nicer that {50} but if you have a better idea ... ;-)

**Review by dinhlongviolin1 — APPROVED** [review]
All good!

**Pull request merged**